File: core/DeleteColumnsToMakeSorted/DeleteColumnsToMakeSorted.java
package com.dbc;

public class DeleteColumnsToMakeSorted {
    public int minDeletionSize(String[] strs) {
        int count = 0;
        for (int col = 0; col < strs[0].length(); col++) {
            for (int row = 0; row < strs.length; row++) {
                if (row != 0 && strs[row].charAt(col) < strs[row - 1].charAt(col)) {
                    count++;
                    break;
                }
            }
        }
        return count;
    }
}

File: core/DeleteColumnsToMakeSorted/DeleteColumnsToMakeSorted.py
from typing import List


def minDeletionSize(strs: List[str]) -> int:
    rows, cols = len(strs), len(strs[0])
    count = 0
    for col in range(cols):
        for row in range(rows):
            if row != 0 and ord(strs[row][col]) < ord(strs[row - 1][col]):
                count += 1
                break

    return count




